src/main/java/com/rakovets/course/java/core/practice/operators/Task05.java

```diff
@@ -34,8 +34,7 @@ public static void main(String[] args) {
      * @param weight - вес
      */
     static int calculateDeviationStandardWeight(int height, int weight) {
-        //TODO
-        // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0;
+        int deviationWeight = weight -(height-110);
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"